Makefile

```diff
@@ -46,6 +46,7 @@ ifeq "${GIT_UPSTREAM_REMOTE}" ""
 GIT_UPSTREAM_REMOTE = "ci-upstream"
 $(warning Could not find git remote for maistra/istio-operator, adding as '${GIT_UPSTREAM_REMOTE}')
 $(shell git remote add ${GIT_UPSTREAM_REMOTE} https://github.com/maistra/istio-operator.git)
+$(shell git config --global --add safe.directory /go/src/github.com/maistra/istio-operator)
```

Comment:
This won't work in a local build. This is meant to workaround an issue in openshift prow.

Reply:
Perhaps using `'*'` instead of hardcoding the path.